### ISSUE TITLE
MD060 opt-in not triggered on format, `extend-enable = ["MD060"]` not working either

### ISSUE THREAD

**Issue by o-l-a-v** [rvben/rumdl-vscode#108]
## Environment 
 
* Windows 11 25H2 x64
* VSCode x64 v1.109.5
* rumdl VSCode extension v0.0.181
* rumdl v0.1.35 installed with Scoop, available in PATH

.rumdl.toml in project root:

```toml
# rumdl configuration
# Schema: https://raw.githubusercontent.com/rvben/rumdl/main/rumdl.schema.json

# Global settings
[global]
disable = ["MD013", "MD025", "MD041"]

# MD003 / heading-style - Heading style should be consistent
[MD003]
style = "atx"

# MD004 / ul-style - Unordered list style should be consistent
[MD004]
style = "asterisk"

# MD029 / ol-prefix - Use consistent numbers for ordered lists
[MD029]
style = "one-or-ordered"

# MD033 / no-inline-html - No HTML tags
[MD033]
allowed-elements = ["details", "summary"]

# MD040 / fenced-code-language - Code blocks should have a language specified
[MD040]
style = "disabled"

# MD055 / table-pipe-style - Keep table formatting consistent
[MD055]
style = "leading-and-trailing"

# MD060 / table-format - Table Format
[MD060]
enabled = true
style = "aligned"
```

## Problem

It won't align tables anymore. It detect that `Table columns should be aligned [MD060]`, but doesn't fix it.

```md
# Test

| Column 1 | Column 2 |
|:--|:--|
| Val11 | Val12 |
| Value 2.1 | Value 2.2 |

```

`rumdl fmt <path_to_file>.md` works though.

VSCode output for rumdl says:

```log
[2026-03-02T18:08:21.157Z] [INFO] Format document requested: c:\Users\<redacted>\OneDrive\IT\Code\Various\temp.md
[2026-03-02T18:08:21.162Z] [INFO] No rumdl fixes available for formatting
```

VSCode output for rumdl Language Server says:

```log
[WARN ] [DEPRECATED] [MD060] enabled = true is deprecated. Use `extend-enable = ["MD060"]` in [global] instead.
```

Setting that does not make the rule work either.

Related: <https://github.com/rvben/obsidian-rumdl/issues/15>
